firmware/src/io/kit.cpp

```
@@ -82,6 +92,208 @@ bool find_pcm(const uint8_t* bytes, uint32_t size, const char* path, uint32_t& o
+  bool have_pluck = false;
+  bool have_id = false;
+  bool have_swing = false;
+  bool have_filter = false;
+  bool have_fx = false;
+  bool have_sidechain = false;
+  char* fields[kMaxFields];
+  for (int i = 1; i < count; ++i) {
+    char* line = lines[i];
+    int got = fields_of(line, "id", fields, 1);
+    if (got == 1) {
+      if (!is_kit_id(fields[0]) || !copy_word(fields[0], kit.id, sizeof kit.id)) return false;
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)